Automattic/woocommerce-payments
# Introduce Code-splitting for intl-tel-input component

Fixes #4623

#### Changes proposed in this Pull Request
This PR introduces code-splitting for the intl-tel-input component, which adds 300+kb to the bundle size. With this change, the component code is exported as a separate chunk and loaded only when requested.

Before:
<img width="272" alt="Screenshot 2022-11-28 at 12 25 33" src="https://user-images.githubusercontent.com/8667118/204255477-6d31de2b-5e40-44a7-b001-b200bd54ff74.png">

After:
<img width="272" alt="Screenshot 2022-11-28 at 12 23 56" src="https://user-images.githubusercontent.com/8667118/204255526-cdff33e5-db61-48c4-aa1e-c735073e34ee.png">


#### Testing instructions
- After checking out the branch, run `npm run build`.
- Verify the bundle size is around `788`kb instead of `1,04`mb.
- Verify the intl-tel-input is loaded correctly on `Payments` -> `Card Readers` -> `Recepit Details`.

-------------------


**Post merge**
